unit_tests/test_config.py

```diff
@@ -794,6 +794,41 @@ def test_29_number_of_nodes_per_az_may_not_be_divisable_by_number_of_az_on_k8s(s
         self.assertEqual(conf.get('n_db_nodes'), 3)
         self.assertEqual(conf.get('availability_zone'), 'b,c')
 
+    @pytest.mark.integration
+    def test_30_check_network_config(self):
```

Comment:
let add test for other backends, to make sure we fail any backend beside aws, when we have 2 interfaces defined
and same for the other options, i.e. if `use_dns` is defined, or ipv6, based on where we have it tested. (i.e. aws first)